src/services/measures/attitude/client/attitude.js

```diff
@@ -1,25 +1,85 @@
-tash.namespace('Gauges.Widgets');
+tash.namespace("Gauges.Widgets");
 
```

Comment:
Not to be picky here (:p) and not a problem for this PR but for the future we can avoid changing existing code for enforcing conventions such as "use-only-double/single quotes"... I purposefully avoid this specific convention; sometimes I use single quotes and sometimes doubles, as I see fit at that time. Same goes for spaces between the name of a function and the opening curly brace and so on (I like this freedom :D ) .. :)